### PR TITLE
ENG-4226 merge 7.1

### PR DESCRIPTION
**Pull request by entgigi**

### CODE REVIEW

**Review by firegloves — APPROVED** [review]
LGTM

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -12,28 +12,16 @@ on:
       - release/*
 
 env:
-  ENTANDO_OPT_USE_PPL_TAG: "v1.3.0"
-  ENTANDO_OPT_ENVIRONMENTS: "${{ secrets.ENTANDO_OPT_ENVIRONMENTS }}"
+  ENTANDO_OPT_USE_PPL_TAG: "v1.4.1"
```
Comment (firegloves) [inline .github/workflows/pr.yml:15]: We need to update also post-merge.yml and ga-publication.yml

**Review by firegloves — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (entgigi) [reply .github/workflows/pr.yml:15]: done

**Review by firegloves — APPROVED** [review]
LGTM

**Review by user-54dbb57a — APPROVED** [review]

**Pull request merged**